presentation-compiler/src/main/scala/org/alephium/ralph/lsp/pc/PCSearcher.scala

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Alephium
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package org.alephium.ralph.lsp.pc
+
+import org.alephium.ralph.lsp.pc.search.CodeProvider
+import org.alephium.ralph.lsp.pc.sourcecode.SourceLocation
+import org.alephium.ralph.lsp.pc.workspace.WorkspaceState
+import org.alephium.ralph.lsp.utils.URIUtil.isFileScheme
+import org.alephium.ralph.lsp.utils.log.{ClientLogger, StrictImplicitLogging}
+import org.alephium.ralph.lsp.utils.IsCancelled
+
+import java.net.URI
+
+object PCSearcher extends StrictImplicitLogging {
+
+  /**
+   * Executes Go-to services.
+   *
+   * @param fileURI        URI of the file where this request was executed.
+   * @param line           Line number within the file where this request was executed.
+   * @param character      Character number within the Line where this request was executed.
+   * @param searchSettings Settings defined for the input [[CodeProvider]].
+   * @param isCancelled  Cancellation support instance.
```

Comment:
:nail_care: `Cancellation support instance` not aligned 🤪

Reply:
This is an older commit. A later one has it formatted.